examples/integration/widgets/data_table_formatting.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
```

Comment:
New examples like this should go under `examples/interaction/widgets/`. `examples/integration` are legacy integration tests, which are currently handled in bokehjs' test suite.